### PR TITLE
[add] user sign up route

### PR DESCRIPTION
**Pull request by judeokafor**

### CODE REVIEW

**Comment by enyeInc** [comment]
@judeokafor remove the package-lock.json from this PR. add it to `.gitignore` file

**Comment by enyeInc** [comment]
@judeokafor This looks good, can you pull in the eslint file from the project template. I see some minor spacing issues here. You can make edits to the eslint file and remove the lints that you do not need for this backend

**Review by enyeInc — APPROVED** [review]

File: functions/src/config/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		private_key_id: process.env.PRIVATE_KEY_ID,
+		private_key: process.env.PRIVATE_KEY,
+		client_email: process.env.CLIENT_EMAIL,
+		client_id: process.env.CLIENT_ID,
+		auth_uri: process.env.AUTH_URI,
+		token_uri: process.env.TOKEN_URI,
+		auth_provider_x509_cert_url: process.env.AUTH_PROVIDER_x509_CERT_URL,
+		client_x509_cert_url: process.env.CLIENT_x509_CERT_URL,
+	},
+};
```
Comment (Simpcyclassy) [inline functions/src/config/index.js:32]: I think @codemutatio already created a config to handle all environment variables, so there won't be conflicts

**Comment by codemutatio** [comment]
This looks good to me

**Review by codemutatio — APPROVED** [review]
This looks good to me and is cool to merged

**Pull request merged**